src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,45 @@
+package calculator;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class StringCalculator {
+    private static Pattern p = Pattern.compile("^[0-9]+$");
+
+    public int add(String text) {
+        String[] splitString;
+
+        if (text == null || text.isEmpty()) {
+            return 0;
+        }
+
+        Matcher m = Pattern.compile("//(.)\n(.*)").matcher(text);
+        if (m.find()) {
+            String customDelimeter = m.group(1);
+            splitString = m.group(2).split(customDelimeter);
+            return sumNumber(splitString);
+        }
+
+        splitString = text.split(":|,");
+        return sumNumber(splitString);
```

Comment:
```suggestion
        if (m.find()) {
            String customDelimeter = m.group(1);
            return sumNumber(m.group(2).split(customDelimeter));
        }
        return sumNumber(text.split(":|,"));
```

꼭 변수에 초기화하지 않아도 메소드에 직접 전달하는 것도 하나의 방법이 될 수 있을 것 같습니다  
어떻게 생각하시나요? 🤔